docs: more Svelte 5 conversion (#12871)
sveltejs/kit: documentation/docs/20-core-concepts/30-form-actions.md

{/if}
```

> [!LEGACY] In Svelte 4

> > In Svelte 4, you'd use `export let form` instead to declare a property

Reviewer: ```suggestion
> In Svelte 4, you'd use `export let data` and `export let form` instead
```
